src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,301 @@
+package duke;
+
+import duke.exception.AddDeadlineException;
+import duke.exception.AddEventException;
+import duke.exception.EmptyNameException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
```

Comment:
Duke class seems to be doing alot of things. consider shifting functions which do the processing and fileIO to classes of their own for a more readable and cleaner code

Reply:
I agree, thanks for the tips!